src/exam/Exam.jsx

```diff
@@ -23,10 +23,12 @@ import messages from './messages';
 const Exam = ({
   isGated, isTimeLimited, originalUserIsStaff, canAccessProctoredExams, children, intl,
 }) => {
-  const state = useContext(ExamStateContext);
   const {
-    isLoading, showTimer, exam, apiErrorMsg, getProctoringSettings,
-  } = state;
+    isLoading, activeAttempt, exam, apiErrorMsg,
+  } = useSelector(state => state.specialExams);
+  const dispatch = useDispatch();
+
+  const showTimer = !!(activeAttempt && IS_STARTED_STATUS(activeAttempt.attempt_status));
```

Comment:
Nit. If you wanted to reduce some of the duplication in the definition of `showTimer`, you could put it in a custom hook.